WordPress/src/main/res/layout/reader_fullscreen_error_with_retry.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/error_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/error_retry"
+        style="@style/WordPress.Button.Primary"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_extra_extra_medium_large"
+        android:layout_marginStart="@dimen/margin_extra_extra_medium_large"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:text="@string/retry"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/error_title" />
+
+    <org.wordpress.android.widgets.WPTextView
```

Comment:
The new stuff we've been doing has used `com.google.android.material.textview.MaterialTextView` instead of WPTextView + we've been moving colors and appearance to `styles`.  Not 100% necc, but may be a good idea to be consistent. Thoughts?

Reply:
>The new stuff we've been doing has used com.google.android.material.textview.MaterialTextView instead of WPTextView 

Good catch! 

> we've been moving colors and appearance to styles. Not 100% necc, but may be a good idea to be consistent. Thoughts?

:P Tbh I'm not a fan of using styles unless the styles are used at more than +- 3 places. Imo it just makes the code less readable. I'm not saying styles shouldn't be used at all, but with our approach to design - each designer has a bit different style and the sections of the app aren't consistent, I don't see many benefits.
Having said that I exported the styles in reader_fullscreen_error_with_retry to reader_styles.xml for consistency. We can discuss this further on slack - perhaps there are some advantages I'm not seeing.

Both fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/12748/commits/695a3daf95d6319f16a7b0a61b93f14150e54eda

Reply:
I see your point about more than 3 or more places. Sounds like we may need a discussion on what consistency means - at the very least we will be consistent within the reader section. Thanks.